src/microbe_stage/MicrobeCamera.cs

```diff
@@ -225,6 +238,17 @@ public void UpdateCameraPosition(double delta, Vector3? followedObject)
                     + currentCameraHeight.Lerp(newCameraHeight, InterpolateZoomSpeed);
             }
 
+            // Apply cursor-induced tilt
+            if (Settings.Instance.MicrobeCameraTilt)
+            {
+                var tilt = new Vector3(cursorVisualWorldPos.X - Position.X, 0, cursorVisualWorldPos.Z - Position.Z) *
+                CursorTiltAmplitude;
```

Comment:
```suggestion
                    CursorTiltAmplitude;
```

Doesn't this need to be intended as this is a line continuation?

Other than this I think this is now correct from a technical standpoint. Though I'm not sure if this should be in the 0.8.2.1 patch as this is kind of a significant change (or if this PR should wait until next week).